app/src/layouts/locationHeaderLayout/userPageLocationLevel/popoverContent/popoverContent.scss

```
@@ -0,0 +1,51 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.crumb {
+  display: flex;
+  align-items: center;
+  gap: 4px;
+  line-height: 20px;
+  word-break: break-word;
```

Stylelint (17.5.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)